src/sentry/integrations/slack/message_builder/notifications/daily_summary.py

```diff
@@ -34,14 +40,34 @@ def __init__(
     def linkify_error_title(self, group):
         link = get_title_link(group, None, False, False, self.notification, ExternalProviders.SLACK)
         title = build_attachment_title(group)
-        return f"<{link}|*{escape_slack_text(title)}*>"
+        attachment_text = self.get_attachment_text(group)
+        if not attachment_text:
+            return f"<{link}|*{escape_slack_text(title)}*>"
+        return f"<{link}|*{escape_slack_text(title)}*> {attachment_text}"
 
     def linkify_release(self, release, organization):
         path = f"/releases/{release.version}/"
         url = organization.absolute_url(path)
         release_description = parse_release(release.version).get("description")
         return f":rocket: *<{url}|Release {release_description}>*\n"
 
+    def get_attachment_text(self, group):
+        attachment_text = build_attachment_text(group)
+        if attachment_text and len(attachment_text) > 50:
+            attachment_text = attachment_text[0:49] + "..."
+        return attachment_text
+
+    def build_discover_url(self, project):
+        query_params = "?"
+        for field in ["title", "event.type", "project", "user.display", "timestamp"]:
+            query_params += f"field={field}&"
+        query_params += "name=All+Events"
+        query_params += f"&project={project.id}&query=event.type%3Aerror&sort=-timestamp&statsPeriod=24h&yAxis=count%28%29"
```

Comment:
idk if there is a more elegant way of doing this, I couldn't find any examples of generating a Discover link or any link with a lot of query params

Reply:
For testing we convert a dict to a query string to add to the URL. The only caveat is it can't take in encoded query param values (like %3A)
https://github.com/getsentry/sentry/blob/09853ff60e695b2f169bcdb29a1d52c7ce27d10f/src/sentry/testutils/cases.py#L706-L708